packages/runtime/package.json

```diff
@@ -52,7 +52,9 @@
     "publish:install": "npm ci",
     "publish:test": "cd .. && npm ci && npm test",
     "clean": "rimraf lib dist-types",
-    "build": "tsc",
+    "build": "run-s build:*",
+    "build:vendor": "rimraf src/templates/vendor && deno vendor src/templates/edge/shims.js src/templates/edge-shared/next-utils.ts https://deno.land/x/html_rewriter@v0.1.0-pre.17/index.ts --output=src/templates/vendor",
```

Comment:
Any new files that import from deno.land will need to be listed here.

Reply:
Is there a reason we add the `html_rewriter` explicit URL rather than `src/templates/edge-shared/utils.ts` package that references it?

Reply:
For some reason, Deno isn't able to vendor `https://edge.netlify.com` correctly. The file that contains `html_rewriter` also references `https://edge.netlify.com`, though, so referencing the file will break :/

Reply:
Thanks!